spec/features/trainees/edit_publish_course_details_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "publish course details", type: :feature, feature_publish_course_details: true do
+  after do
+    FormStore.clear_all(trainee.id)
```

Comment:
This is required or some responses get saved between tests in redis, should we add this to rails helper as well?